indexer/ethereumetl/mappers/block_mapper.py

```diff
@@ -103,7 +103,7 @@ def schema(cls):
             pa.field('receipts_root', pa.string()),
             pa.field('miner', pa.string()),
             pa.field('difficulty', pa.decimal128(precision=38, scale=0)),
-            pa.field('total_difficulty', pa.decimal128(precision=38, scale=0)),
+            pa.field('total_difficulty', pa.string()),
```

Comment:
can you give an example?

Reply:
Total difficulty of block [https://etherscan.io/block/17614100](url) is more than max size of C long

Reply:
And you statement also wrong, precision 38 can have more than this number, so this not the rootcause